test/php/library/Icinga/Application/CommonPasswordPolicyTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Tests\Icinga\Application;
+
+use Icinga\Application\Hook\PasswordPolicyHook;
+use Icinga\Application\ProvidedHook\CommonPasswordPolicy;
+use Icinga\Test\BaseTestCase;
+
+class CommonPasswordPolicyTest extends BaseTestCase
```

Comment:
You can extend `PHPUnit\Framework\TestCase` instead, then you do not need the previously added `composer.json`. Please also remove the entry from the `.gitignore` file.